mix.exs

```diff
@@ -38,6 +36,20 @@ defmodule ElixirSense.Mixfile do
     ]
   end
 
+  defp dialyzer do
+    [
+      plt_add_deps: :apps_direct,
+      plt_add_apps: [:mix],
```

Comment:
after 7c6e1298e8f703cf7732d6193449f6dafe3bdf3e this is no longer needed

Reply:
awesome, this one can go then 🙂